recipe/activate.sh

```diff
@@ -7,33 +7,57 @@
 # doesn't come with a deactivate script, because the symlink
 # is benign and doesn't need to be deleted.
 
+_la_log() {
+    if [ "$LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
+        echo "${BASH_SOURCE[0]} DEBUG: $*"
+    fi
```

Comment:
We don't get to "steal" arrows namespace here, let's rename the variable to something like:
```suggestion
    if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
        echo "${BASH_SOURCE[0]} DEBUG: $*"
    fi
```